openshift/deploy/job/cas-shelf-tfe-add-app.yml

```diff
@@ -18,7 +18,7 @@ spec:
             requests:
               cpu: 500m
               memory: 512Mi
-          image: docker.pkg.github.com/bcgov/cas-shelf/cas-shelf-tfe-add-app:0.1.0
+          image: docker.pkg.github.com/bcgov/cas-shelf/cas-shelf-tfe-add-app:0.1
```

Comment:
Have we already added a note here on our tagging convention? The removal of patchlevel to refer only to `major.minor` release might not otherwise make sense to a reader.

Reply:
hm.. whenever pushing a new image actually it overrides upper-level tag images as well.
and my intention was here to just ignore patch level and use whatever pushed the latest onto a minor level.
I will add a comment regarding the tag convention though

Reply:
> hm.. whenever pushing a new image actually it overrides upper-level tag images as well.

that totally makes sense, and since we're handing this code over that might not be obvious to another developer without documentation